### PR TITLE
Population projections methodology page styling updates

### LINKED ISSUE #878: Tweak Projections Methodology page
**What needs to be done? Why does it need to be done?**
Update the Projections Methodology Page to be aligned with the new design system and to be easier to read.

**Before/After**

**Changes:**

1. Update text styles to match the [System 2.0.0 Article styles](https://www.figma.com/file/v7ei7OnUkPkwJO6ukOvTjl/System-2.0.0?node-id=1%3A25). The new styles are indicated in the [new mockup](https://www.figma.com/file/SOdY6JujMpbawgJ4jjDmE8/Population-Projection-MVP?node-id=1166%3A0).

2. Increase page margins to increase legibility. The maximum width of the text container should be 1184px. 
 

**Update Text**
To replace the section called ' Confidence Intervals' (remove quotes)

"**Prediction Intervals**
To communicate the confidence of the projections, the Spark model produces prediction intervals that bound the forecast. These prediction intervals are constructed by resampling historical one-period model errors. The model estimates the distribution of model errors at time step t by summing t draws from the historical one-period model errors multiple times with replacement. From this distribution of errors, the model calculates the width of the prediction by subtracting the 2.5th percentile model error from the 97.5th percentile model error. Finally, the model centers this 95% prediction interval width on the projection, thereby bounding the forecast."

**Additional context**
Ping @soniachokshi with any questions!

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Updates the styles for the population projections Methodology page. Also includes one small copy change.

I started to go down the road of converting all of the Methodology components to styled-components, but it seemed like it became too unruly and did not simplify anything so I decided to leave it as is.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #878

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jovergaag** [comment]
@soniachokshi and @chrispmlee The review app is [here](https://recidiviz-dashboard-stag-e1108--methodology-co6ea0yh.web.app/methodology) for you to QA! LMK if there are any other little tweaks you'd like to see.

**Comment by chrispmlee** [comment]
This looks great! I love being able to click through each section. One small thing is the size of the font for the first section looks really big. @soniachokshi do you have any recommendations on font size?

File: src/core/CoreConstants.scss [hunk 1]
```diff
@@ -6,6 +6,18 @@ $core-breakpoint-sm: 1024px; /* iPad Pro */
...
+$slate-80: rgba(53, 83, 98, 0.8);
+// Shorthand font property: font-style font-variant font-weight font-size/line-height font-family
+$font-ui-sans-16: 500 1rem/1.5rem Libre Franklin;
+$font-ui-sans-14: 500 0.875rem/1.5rem Libre Franklin;
+$font-article-serif-56: 400 3.5rem/4.5rem Libre Baskerville;
+$font-article-serif-34: 400 2.1rem/3rem Libre Baskerville;
+$font-article-serif-24: 400 1.5rem/2rem Libre Baskerville;
+$font-article-sans-32: 400 2rem/3rem Libre Franklin;
+$font-article-sans-16: 400 1rem/1.7rem Libre Franklin;
+$font-article-sans-14: 400 0.88rem/1.5rem Libre Franklin;
```
Comment (Catacola) [inline src/core/CoreConstants.scss:20]: There is a section for font variables starting on L64. The first two variables on L13 and L14 are already there. Plue we have the `$sand-serif` and `$serif` variables to make font-switching easier in the future.

File: src/core/CoreConstants.scss [hunk 2]
```diff
@@ -6,6 +6,18 @@ $core-breakpoint-sm: 1024px; /* iPad Pro */
 $core-breakpoint-xs: 768px; /* iPad */
 
 $core-max-view-width: 93rem;
+$core-font-color: #00413e;
+$pine-1: rgba(1, 35, 34, 1);
+$slate-80: rgba(53, 83, 98, 0.8);
```
Comment (Catacola) [inline src/core/CoreConstants.scss:11]: `$core-font-color` is the same as `$pine-3` on L37 below.
`$pine-1` and `$slate-80` are defined on L35 and L48, respectively.

**Review by Catacola — APPROVED** [review]
Thanks for doing this! All looks good from the code side save a few minor nits about variable grouping.

File: src/core/CoreConstants.scss (hunk 2 above)
Comment (jovergaag) [reply src/core/CoreConstants.scss:11]: I think all of these redundancies were from a sloppy merge. Thanks for catching them!

**Comment by jovergaag** [comment]
@chrispmlee @soniachokshi I made the changes from the design QA - ready for another look!
https://recidiviz-dashboard-stag-e1108--methodology-co6ea0yh.web.app

I think it looks much better with that first paragraph sized down...good call.

**Comment by soniachokshi** [comment]
Woohoo! Looks great – thanks for making those tweaks, Jen! 

There are only a couple of last little things I noticed:


The line-height should be 40px / 2.5rem here instead of 2rem


Can we reduce letter spacing for the H4 ("Compartments" in this example) to -0.04em as well? (My bad, I totally missed that on the last go around.)


And can we add 72px of padding between the last paragraph and the footer?

That's all!

**Pull request merged**